html/editevent.js

```
@@ -0,0 +1,59 @@
+/* global EasyMDE */
+
+document.addEventListener("DOMContentLoaded", (event) => {
```

ESLint

[error] 'event' is defined but never used.

(no-unused-vars)